samples/docker-domain/container-scripts/domain.properties

```diff
@@ -0,0 +1,2 @@
+ADMIN_PORT=7001
```

Comment:
Why do you need this domain.properties if you have a simple-properties.yaml and can define the defaults  in the Dockerfile

Reply:
We don't currently use this for the admin startup script, not sure if we will need for the server scripts. One problem is that we can't easily extract the values we want from the yaml model file. I'll know more once we have startup scripts.